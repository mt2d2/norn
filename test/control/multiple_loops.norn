def main(): Void
  i: Int = 0

  for i: Int = 0, i < 10000000, i=i+1 in
    i = i +1
  end

  println(i)

  i = 0

  for i: Int = 0, i < 8888, i=i+1 in
    i = i +1
  end

  println(i)
end
