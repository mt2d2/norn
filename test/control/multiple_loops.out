10000000
8888
